### ISSUE TITLE
Toast: close button missing aria-label

### ISSUE THREAD

**Issue by tlframe2** [primefaces/primevue#2825]
### Describe the bug

Close button on toasts lack an aria-label which is important since the content of the button is just an icon
![toast](https://user-images.githubusercontent.com/32079409/182683059-28e00daf-1643-471b-a9b3-ca482a144c09.png)

### PrimeVue version

3.15.0

### Vue version

3.x

### Language

TypeScript

### Build / Runtime

Vue CLI App

**Comment by tugcekucukoglu** [comment]
It was added to Toast component. Please try to use the latest PrimeVue version.
